index.js

```diff
@@ -115,7 +115,13 @@ class SideMenu extends Component {
    */
   handlePanResponderMove(e: Object, gestureState: Object) {
     if (this.state.left.__getValue() * this.menuPositionMultiplier() >= 0) {
-      this.state.left.setValue(this.prevLeft + gestureState.dx);
+      let newLeft = this.prevLeft + gestureState.dx;
+
+      if (!this.props.bounceBackOnOverdraw && Math.abs(newLeft) > this.props.openMenuOffset) {
```

Comment:
I don't like the idea of implementing something "just because I can". Can you please give me a real example when it's needed?